tools/spelling/spelling_dictionary.txt

```diff
@@ -201,53 +188,34 @@ NOLINT
 NOLINTNEXTLINE
 NONBLOCK
 NONCES
-NOSORT
 NS
 NUL
 Nilsson
-Nonhashable
-Oauth
 OCSP
-OID
 OK
 OOM
-OOMs
 OS
 OSI
 OSS
-OSX
 OT
 OU
 OVFL
 PAYLOADLEN
 PB
-PCC
-PEERCRED
 PEM
 PERF
 PGV
-PID
 PKTINFO
-PNG
-PostCBs
 PREBIND
 PRNG
 PROT
-Postgre
 Postgres
-Prereq
 QDCOUNT
 QUIC
-QoS
```

Comment:
QoS is widely understood and should be left

Reply:
Done.